pkg/shared/util/volume.go

```diff
@@ -30,9 +30,20 @@ var (
 	configMapKeySelectorType = reflect.TypeOf(&corev1.ConfigMapKeySelector{})
 )
 
+type GetSecrets interface {
```

Comment:
1. I assume the purpose of the interface is to be able to mock it for testing? If that's the case, make it and `OsFile`?
2. The name does not match the 2 functions.

Reply:
> I assume the purpose of the interface is to be able to mock it for testing?
yes I added that as a comment 

> If that's the case, make it and OsFile?
Sorry, I don't understand, what you mean by that

> The name does not match the 2 functions.
I renamed to `volumeReader`, what do you think?